README.md

```diff
@@ -49,9 +75,11 @@ This variable is be used to determine whether or not the libraries will be compi
   * `LibrarySONAME` and `LibraryFull` will both be set to `$(@F)`, and no symlinks will be generated
 
 ##### Package structure variables
-* `INSTALL_PATH` is the base directory of the installed project, defaults to `/opt/$(Project)`
-* `ProjectPath` is the base directory of the project, defaults to `$(BUILD_HOME)/$(Project)`
-* `PackagePath` is the base directory of the package, defaults to `$(ProjectPath)`
+* `INSTALL_PREFIX` : where to install the project, defaults to `/`, may be overridden for non-system installation
+* `INSTALL_PATH` : base directory of the installed project, defaults to `/opt/$(Project)`, **should not be overridden**
+* `PETA_PATH` : base directory where cross-compilation libraries will be installed on the host, defaults to `/opt/gem-peta-stage`, **should not be overridden**
```

Comment:
Why shouldn't `PETA_PATH` be overridden? Isn't it supposed the base directory for all boards filesystem roots?

Reply:
Because both `PETA_PATH` and `INSTALL_PATH` are intentionally designed to *only* provide a fixed directory structure within the package, while `INSTALL_PREFIX` is the thing that gives freedom of where to put this predefined package structure.
If you have a suggestion on wording to make this more clear, please do

Reply:
Ok, it makes senses. I currently use `PETA_PATH` as `PETA_STAGE` because... it works.